src/api-reference/user-provisioning/v4.user-provisioning.md

```diff
@@ -7,17 +7,55 @@ layout: reference
 
 {% include prerelease.html %}
 
-The User Provisioning Service allows callers to provision a user in the SAP Concur environment. Once a user is provisioned, the user profile is set up in Identity, Travel, and Spend.
+The User Provisioning Service allows callers to provision and update a user’s profile in the SAP Concur environment. Once a user is provisioned, the users profile is available to support the Identity, Travel and Spend services. The service allows clients and partners to modify profiles in bulk or individually using standard REST verbs to create, update objects following a defined schemas covering a users identity, spend and travel data attritubues. 
+
+* [Limitations](#limitations)
```

Comment:
Please remove this navigation list. The site automatically generates a nav on the right hand side of the page now.

Reply:
Removed.